website/docs/reference/global-configs/warnings.md

```diff
@@ -73,14 +72,27 @@ The `error` parameter can be set to `"all"` or `"*"` to treat all warnings as er
 - When `error` is set to `"all"` or `"*"`, the optional `warn` parameter can be set to exclude specific warnings from being treated as exceptions.
 - Use the `silence` parameter to ignore warnings. To silence certain warnings you want to ignore, you can specify them in the `silence` parameter. This is useful in large projects where certain warnings aren't critical and can be ignored to keep the noise low and logs clean.
 
-Here's how you can use [`--warn-error-options`](#use---warn-error-options-for-targeted-warnings) flag to promote _specific_ warnings to errors:
+Here's how you can use the [`--warn-error-options`](#use---warn-error-options-for-targeted-warnings) flag to promote _specific_ warnings to errors:
 - [Test warnings](/reference/resource-configs/severity) with the `--warn-error-options '{"error": ["LogTestResults"]}'` flag.
 - Jinja [exception warnings](/reference/dbt-jinja-functions/exceptions#warn) with `--warn-error-options '{"error": ["JinjaLogWarning"]}'`.
 - No nodes selected with `--warn-error-options '{"error": ["NoNodesForSelectionCriteria"]}'`.
 - Adapter deprecation warnings with `--warn-error-options '{"error": ["AdapterDeprecationWarning"]}'`.
 
 </VersionBlock>
 
+<VersionBlock firstVersion="1.10">
+
+### Set deprecation warnings as errors
+
+You can use the [`--warn-error-options`](#use---warn-error-options-for-targeted-warnings) flag to promote _deprecation_ warnings to errors, for example:
+
+- Set only deprecation warnings as errors with `--warn-error-options="{'error': ['deprecations']}"`
```

Comment:
same comment re: capitalization of `Deprecations`

Reply:
➕